CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+<a name="v1.124.1"></a>
+## v1.124.1 (2018-11-07)
+
+
+#### Bug Fixes
+
+* **mime:**  use base64 encoding instead of 8bit ([5d03a48d](5d03a48d))
```

Comment:
did we miss out on the rest of the changes here?

Reply:
Yeah, the changelog script for this repo doesn't pick up `chore`. I plan to do something about it as part of #214.

Reply:
yea clog doesn't like chores

Reply:
And `refactor` too I guess.